Improve template of command service gen doc (#617)
mesg-foundation/core: commands/provider/assets/readme_template.md

# {{.Name}}

{{.Description}}

# Contents

- [Installation](#Installation)
- [Definitions](#Definitions)
  {{if .Events}}- [Events](#Events){{range $key, $event := .Events}}
    - [{{or $event.Name $key}}](#{{or $event.Name $key | anchorEncode}}){{end}}{{end}}
  {{if .Tasks}}- [Tasks](#Tasks){{range $key, $task := .Tasks}}
    - [{{or $task.Name $key}}](#{{or $task.Name $key | anchorEncode}}){{end}}{{end}}

# Installation

## MESG Core

> This service requires [MESG Core](https://github.com/mesg-foundation/core) to be install.

Reviewer: ```suggestion
This service requires [MESG Core](https://github.com/mesg-foundation/core) to be installed first.
```

> You can install MESG Core by running the following command or [follow the install guide](https://docs.mesg.tech/guide/start-here/installation.html).

Reviewer: ```suggestion
You can install MESG Core by running the following command or [follow the installation guide](https://docs.mesg.tech/guide/start-here/installation.html).
```

```bash
bash <(curl -fsSL https://mesg.com/install)
```

## Service

{{if .Repository}}To deploy this service, run the following command:
```bash
mesg-core service deploy {{.Repository}}
```
{{else}}Download the source code of this service, and then in the service's folder, run the following command:
```bash

> mesg-core service deploy ./

Reviewer 2: ```suggestion
mesg-core service deploy
```
